cli.js

```diff
@@ -15,6 +15,10 @@ let asyncapi;
 let template;
 
 const parseOutput = dir => path.resolve(dir);
+const parseJSON = value => {
+  try { return JSON.parse(value); }
+  catch (e) { return {}; }
```

Comment:
I love the idea of "not breaking" and continue with normal execution using a default value of `{}`. What do you think about putting a warning here? I'm thinking in the case people make a mistake in their JSON string. They would not understand what's happening.

Reply:
You're right. Maybe adding something like:
```js
console.log(yellow(`Warning: values of --params will not be available in the templates, there was a error parsing: ${value}`));
console.log(magenta(e.message));
```
If you agree, I can update the pull request

Reply:
Looks good to me 👍